### DISCUSSION TITLE
Credit Invoice not Visually Distinct

### DISCUSSION THREAD

**Discussion by sbts** [ledgersmb/LedgerSMB#7850] [Ideas]
Currently the only visual difference between a Sales Invoice and a Sales Credit Invoice is the wording of the Heading.
- `Sales Invoice`
- `Credit Invoice`

There is absolutely no other difference between the two pages.
This is most noticeable if you have just used the `VOID` button on a `Sales Invoice`
You are returned to a screen that is IDENTICAL except for the title, which is very easily missed.

The user then thinks they have not actually generated a void invoice as the QTY's and `$` are exactly the same as the original invoice.

It may also be worth considering displaying Negative values for
- QTY's
- Line Totals
- Subtotal
- Taxes
- Total

You would continue to show positive values for unit price though

**Comment by ehuelsmann** [comment]
Duplicate of #2280

**Comment by freelock** [comment]
I'm wondering if we can get a designer involved here to define a design language for these?

I'm thinking we should have a large icon somewhere consistent on the screen that distinguishes invoices, transactions, GL entries, orders, quotes at a glance, and perhaps a color scheme change between AR and AP -- and a secondary color change for "exception" types like credit/debit invoices/notes. Not sure how to make that accessible for the colorblind...

**Comment by ehuelsmann** [comment]
The fact that the two show exactly the same numbers is by design. When we showed negative numbers, people complained negative credit invoices can be confused to be regular invoices (negating the credit...l
